### ISSUE TITLE
Support TestDox

### ISSUE THREAD

**Issue by fntlnz** [facile-it/paraunit#25]
I think that would be nice if paraunit can generate the PHPUnit testdox in parallel, what do you think?

**Comment by Jean85** [comment]
Could be a nice to have! I'll have to take a look to PHPUnit classes to decide about it, I need to see how much of them I can reuse to do that.

**Comment by ranpafin** [comment]
Just a few ideas here:

*Testdox* usually output something like: 
Foo
- output foo when the input is baz
- throws an exception when the input is empty
- [...]

I think the best way to go here is to create a parser (and a printer) much like the one for the 'standard output'.
This new parser could be load using the new dynamic event structure and will replace the classic one. The behavior would be pretty straightforward just execute all the phpunit processes with the testdox argument ,read the output merge it in the final result.
Usually what you may want to do with *testdox* is output a text-file with all the method inside, luckily we already accumulate all the output so creating such file should be trivial.
While one parser would be enought there should be 2 printer a 'console' printer and a 'text-file' printer.

**Comment by Jean85** [comment]
This seems to have no interest. Since the, we moved to the JSON log format for parsing, and we got performance improvements from that.

Going back to output parsing seems a step backward for me.
